ui/src/components/facets/HistogramFacet.js

```diff
@@ -88,6 +95,35 @@ class FacetHistogram extends Component {
       !this.props.selectedValues.includes(facetValue.name)
     );
   }
+
+  onClick(event, item) {
+    // Ignore clicks which are not located on histogram
+    // bars.
+    if (item != null) {
```

Comment:
```
if (item && item.datum) {
```
If I click inside the graph area but not on a bar, item is set and item.datum is not set.

Reply:
Done.